### PR TITLE
create modeling rules on google drive

### PR DESCRIPTION
**Pull request by guytamir10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc review completed.

File: Packs/GoogleDrive/ModelingRules/GoogleDrive_1_3/GoogleDrive_1_3.xif [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[MODEL: dataset = google_drive_raw]
```
Comment (evisochek) [inline Packs/GoogleDrive/ModelingRules/GoogleDrive_1_3/GoogleDrive_1_3.xif:1]: Dataset name is not correct

File: Packs/GoogleDrive/ModelingRules/GoogleDrive_1_3/GoogleDrive_1_3_schema.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "google_drive_raw": {
```
Comment (evisochek) [inline Packs/GoogleDrive/ModelingRules/GoogleDrive_1_3/GoogleDrive_1_3_schema.json:2]: Dataset name is not correct

File: Packs/GoogleDrive/ParsingRules/GoogleDrive/GoogleDrive.xif [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[INGEST:vendor="google", product="drive", target_dataset="google_drive_raw", no_hit = keep]
```
Comment (evisochek) [inline Packs/GoogleDrive/ParsingRules/GoogleDrive/GoogleDrive.xif:1]: Dataset name is not correct

**Review by evisochek — APPROVED** [review]

**Pull request merged**